MetaMask/metamask-extension
# chore: add privacy query params to portfolio navigation

## **Description**

- Added `metricsEnabled` and `marketingEnabled` query params to reflect user privacy preferences on user navigation to Portfolio. These settings, sent from extension, will be consumed by the Portfolio to update and align privacy preferences.

- Updated all relevant e2e and unit tests to include the params. 

Context: Our current privacy opt-in rate has been low since some recent updates to our privacy flow, so we are aligning our privacy settings with the other platforms.

## **Related issues**
Fixes:

## **Manual testing steps**

1. Update privacy preferences in settings for both MetaMetrics and Marketing
2. Navigate to Portfolio 
4. See that the privacy params accurately reflect the user's settings in mobile

Example URL:

`https://portfolio.metamask.io/?metamaskEntry=ext_portfolio_button&metametricsId=0xbcd1a47f61d820aac3158af775dfee88828e5330aae11f3048767aab7e99b474&metricsEnabled=false&marketingEnabled=false`

## **Screenshots/Recordings**



### **Before**



### **After**


With query params attached:
https://www.loom.com/share/7f372324de1e452a837e5faa9860ee94

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
